vulnfeeds/cves/versions_test.go

```diff
@@ -631,20 +707,26 @@ func TestExtractGitCommit(t *testing.T) {
 				Repo:  "https://github.com/eggjs/extend2",
 				Fixed: "aa332a59116c8398976434b57ea477c6823054f8",
 			},
+			disableExpiryDate: NO_EXPIRY,
 		},
 		{
 			description:            "A GitHub commit link that is 404'ing (as seen on CVE-2019-8375)",
 			inputLink:              "https://github.com/WebKit/webkit/commit/6f9b511a115311b13c06eb58038ddc2c78da5531",
 			inputCommitType:        Fixed,
 			expectedAffectedCommit: AffectedCommit{},
 			expectFailure:          true,
+			disableExpiryDate:      NO_EXPIRY,
 		},
 	}
 
 	for _, tc := range tests {
 		if _, ok := os.LookupEnv("BUILD_ID"); ok && tc.skipOnCloudBuild {
 			t.Skipf("test %q: running on Cloud Build", tc.description)
 		}
+		if time.Now().Before(tc.disableExpiryDate) {
+			println(tc.description)
```

Comment:
You want the `t.Logf` function here, and probably a more descriptive log, e.g. `Skipping test: %s`